Mathlib/Data/Multiset/ZeroCons.lean

```diff
@@ -357,7 +357,7 @@ theorem le_cons_self (s : Multiset α) (a : α) : s ≤ a ::ₘ s :=
 theorem cons_le_cons_iff (a : α) : a ::ₘ s ≤ a ::ₘ t ↔ s ≤ t :=
   Quotient.inductionOn₂ s t fun _ _ => subperm_cons a
 
-theorem cons_le_cons (a : α) : s ≤ t → a ::ₘ s ≤ a ::ₘ t :=
+@[simp] theorem cons_le_cons (a : α) : s ≤ t → a ::ₘ s ≤ a ::ₘ t :=
```

Comment:
I think it makes more sense to tag `cons_le_cons_iff` here. If `simp` can prove `s ≤ t`, I guess they'll both work, and otherwise it makes more partial progress.